docs/newbs_getting_started.md

```diff
@@ -42,9 +42,13 @@ We've tried to make QMK as easy to set up as possible. You only have to prepare
 You will need to install MSYS2, Git, and the QMK CLI.
 
 * Follow the installation instructions on the [MSYS2 homepage](http://www.msys2.org).
-* Close any open MSYS2 terminals and open a new MSYS2 MinGW 64-bit terminal.
+* Close any open MSYS2 terminals and open a new MSYS2 MinGW 64-bit terminal. NOTE: This is **not** the same as the MSYS terminal that opens when installation is completed.
 
-After opening a new MSYS2 MinGW 64-bit terminal run these commands:
+After opening a new MSYS2 MinGW 64-bit terminal, make sure `pacman` is up to date with:
+
+    pacman -Syu
```

Comment:
The only problem with doing this, and why we remove it .... is that this will make terminal window hang, intentionally if one or more core components are updated. 

It tells you to close the window, agree to the dire warning, and reopen. But ... that's a step that has caused a lot of confusion....

Reply:
And that's exactly why it has to be pointed out here, and done first thing, instead of having the install script do it.